qualtricssurvey/private/view.js

```diff
@@ -0,0 +1,11 @@
+function QualtricsSurveyView(runtime, element) {
```

Comment:
Hrmm, it looks like this doesn't even do anything?
Maybe we should just omit this file from the page in the first place
and remove it entirely.